### ISSUE TITLE
Earn: Purchasers making one-time payments receive billing emails with confusing terminology and branding.

### ISSUE THREAD

**Issue by kwight** [Automattic/wp-calypso#44046]
Those that make one-time payments through Earn features will receive a receipt-type email referencing their "one-time subscription", as well as other uses of "subscription". The email is also topped by the Automattic logo, which could be surprising and meaningless to many users (WordPress.com branding, like the newer emails, could be more appropriate).

<img width="742" alt="Screen Shot 2020-07-09 at 9 46 09 AM" src="https://user-images.githubusercontent.com/349751/87069214-ab9df880-c1cb-11ea-95e0-23f12fcc36b1.png">

**Comment by mmtr** [comment]
I don't think we can address this correctly without updating the plans management UI in Calypso.

So far, we are considering that all plans a user can manage in wordpress.com/earn/payments are subscriptions, including the one-time ones (they just happen to be lifetime subscriptions) so the terminology is correct under that assumption.

We actually display this notice when a user tries to purchase/sign up to a one-time twice, in which the "subscription" terminology makes even more sense (the user already paid for a lifetime subscription, they don't need to pay more):

<img width="434" alt="Screen Shot 2020-08-31 at 15 24 47" src="https://user-images.githubusercontent.com/1233880/91724795-1f41fd80-eb9e-11ea-8e97-5f0c60c8627c.png">

The confusion comes when a site does not sell a subscription but a product. For those cases, I think it'll be better to display a selector to explicitly select the type of plan when creating one, so we can adapt the terminology depending on the type (as we currently do with donation-type plans).

**Comment by kristarella** [comment]
+1 on this issue, even if the result is as @mmtr suggested, to make a different type of plan that isn't a "subscription"
